apps/prairielearn/src/schemas/infoAssessment.ts

```diff
@@ -362,6 +362,13 @@ export const AssessmentJsonSchema = z
       .describe('Whether students can create groups.')
       .optional()
       .default(false),
+    studentGroupName: z
```

Comment:
I know I mentioned in my other comment that we're thinking of "name" as a verb... but I actually don't think that's great? I'm wondering if `studentGroupChooseName` might be a little better, or at least more explicit. Then the database column could be named similarly.